### ISSUE TITLE
PHPUnit v8 assertInternalType is deprecated

### ISSUE THREAD

**Issue by donnysim** [kalessil/phpinspectionsea#1292]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 3.0.13 |
| Language level | PHP 7.2                     |

Current behaviour
--
It suggests converting something like 
```
PHPUnit::assertTrue(\is_array($this->data), 'Json data is empty.');
```
into 
```
PHPUnit::assertInternalType('array', $this->data, 'Json data is empty.');
```
But then you end up with `Method assertInternalType is deprecated` warning.

Expected behaviour
--
Suggest methods as stated in the warning:
```
assertInternalType() is deprecated and will be removed in PHPUnit 9. Refactor your test to use assertIsArray(), assertIsBool(), assertIsFloat(), assertIsInt(), assertIsNumeric(), assertIsObject(), assertIsResource(), assertIsString(), assertIsScalar(), assertIsCallable(), or assertIsIterable() instead.
```

Environment details
---

PhpStorm 2019.1.1 Preview
Build #PS-191.6707.42, built on April 10, 2019
JRE: 1.8.0_202-release-1483-b44 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by someniatko** [comment]
I'd disagree with "false-positive" flag. Rather, the quick-fix is incorrect.

**Comment by kalessil** [comment]
Duplicates #1152. @donnysim, we are introducing PhpUnit version setting there, which will take care of proper replacement generation.
